www/layouts/collections/single.html

```diff
@@ -47,7 +47,7 @@ <h1>
             {{- $course_list_data = $course_list_data | append (dict $id $data) -}}
           {{ end }}
         {{ else }}
-          {{ $errorMessage := printf "Failed to fetch collections through %v on %v" $url site.BaseURL }}
+          {{ $errorMessage := printf "Failed to fetch course data from https://ocw.mit.edu/courses/14-04-intermediate-microeconomic-theory-fall-2006/data.json. Skipping course on page %v" site.BaseURL }}
```

Comment:
I don't think we want to hardcode the URL:
```suggestion
          {{ $errorMessage := printf "Failed to fetch course data from $url. Skipping course on page %v" site.BaseURL }}
```
I believe the suggestion in https://github.com/mitodl/ocw-hugo-themes/issues/829#issuecomment-1240616619 was mostly around making it clearer that the course is skipped (and others continue)

Reply:
Done.